### DISCUSSION TITLE
Student Developer Pack approved but still not available after 1 month

### DISCUSSION THREAD

**Discussion by martincazaban** [community/community#74735] [GitHub Education]
### Hi everyone,

I'm not getting access to the Student Developer Pack and never got the confirmation email even though I was approved on october 1st, more than a month ago...

On the GitHub Education page I find contradictions, according to (https://education.github.com/pack#offer) my application is still being processed.

<img width="1334" alt="Captura de Pantalla 2023-11-08 a la(s) 17 36 43" src="https://github.com/community/community/assets/127999341/1d579748-af6c-4d6a-a0c0-777108d7c9fc">

But according to (https://education.github.com/discount_requests/application) it is already approved.

<img width="352" alt="Captura de Pantalla 2023-11-08 a la(s) 17 25 55" src="https://github.com/community/community/assets/127999341/8c3ae42a-b4f2-4312-9021-2bc16e1ea454">

When I try to resubmit my information to verify that I am a student, I am told that I am already verified.

<img width="601" alt="Captura de Pantalla 2023-11-08 a la(s) 17 55 03" src="https://github.com/community/community/assets/127999341/cd752e81-e2c8-4a1f-9c83-6171f07cd356">

My GitHub user profile does not update with the PRO logo. I've read a lot similar discussions but I can't find this specific situation or any solution.

**Comment by gitstua** [comment]
Hi @martincazaban

Thanks for the being part of our community!

I have asked the relevant team to check in on this for you.

Normally the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them if you have not already :)

**Reply by gitstua** [reply]
Hi @martincazaban 

I have worked with the team and if you can just go through and re-apply  this will hopefully resolve your issue and get a new coupon.

Given we are close to a weekend and it is our GitHub Universe event this week you may need to wait at least a few days after you re-apply.

Sorry for the issue you have been experiencing here.

**Reply by martincazaban** [reply]
Hi @gitstua 

I have done what you indicated and now I have access to the Pack, thank you very much for your help and time!

**Comment by dkhak3** [comment]
thank sir

**Comment by dkhak3** [comment]
i need it
